### DISCUSSION TITLE
Revert to default dialog in the skin editor should apply "underwater effect" to beatmap hitsounds

### DISCUSSION THREAD

**Discussion by Loreos7** [ppy/osu#22835] [Q&A] [answered]
Version: current master
This dialog window should dim the volume for everything on the background: music and hitsounds.

https://user-images.githubusercontent.com/86934170/224479701-b34a457a-ed84-40ad-bdec-ac1811e523cc.mp4

**Comment by Joehuu** [comment, marked as answer]
This affects all dialogs, so marking this the same as https://github.com/ppy/osu/discussions/22053.
